src/QsCompiler/Tests.Compiler/TestCases/QirTests/ExecutionTests.qs

```diff
@@ -1651,6 +1651,26 @@
     }
 
 
+    @EntryPoint()
+    operation TestDefaultValues() : Unit {
+        Message($"{new Int[3]}");
+        Message($"{new Double[3]}");
+        Message($"{new Bool[3]}");
+        Message($"{new Pauli[3]}");
+        Message($"{new String[3]}");
+        Message($"{new Range[3]}");
+        Message($"{new Qubit[3]}");
+        Message($"{new Result[3]}");
+        Message($"{new Unit[3]}");
+        // Todo: bigint is not yet supported in the runtime
```

Comment:
FYI: One of the best practices recommends for every "Todo" to provide either the alias or the bug number, e.g.
"Todo(**bettinaheim**): ..." or
"Todo(**\<bug number\>**): ...".